### PR TITLE
Lazily batch fetching of missing users

### PR DESCRIPTION
**Pull request by callum-oakley**
Collect fetch missing user requests and fire them off at most every 100ms.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@pusher/chatkit-client",
   "description": "Pusher Chatkit client SDK for browsers and react native",
   "main": "dist/web/chatkit.js",
-  "version": "1.13.1",
+  "version": "1.13.2-beta.1",
```
Comment (jonathanlloyd) [inline package.json]: Is this meant to be here?

Comment (callum-oakley) [reply package.json]: no harm in it. It'll be changed after merging when we cut a non-beta release.

File: src/user-store.js [hunk 2]
```diff
@@ -78,3 +78,63 @@ export class UserStore {
     return basicUser ? new User(basicUser, this.presenceStore) : undefined
   }
 }
+
+function batch(f, maxWait, maxPending) {
```
Comment (jonathanlloyd) [inline src/user-store.js]: No unit tests for batch?

**Comment by callum-oakley** [comment]
@jonathanlloyd added some tests

**Comment by callum-oakley** [comment]
taking "otherwise LGTM" as approval. Would still appreciate some eyes on the tests and the comments, but I'd like to push on with #213 :)

**Pull request merged**